tests/test_full_text_search.py

```diff
@@ -192,6 +202,12 @@ def test_search_by_name_only(self):
         assert set([r.record_id for r in results]) == \
             set(['haiti/0225'])
 
+        # Search by full name without space
```

Comment:
Also add a unit test to test full name without space + romanization applied (both unicode one and Japanese dictionary one). My guess is that the test fails :)